inference-engine/tests_deprecated/functional/vpu/common/layers/myriad_layers_resample_test.hpp

```diff
@@ -126,6 +139,8 @@ TEST_P(myriadResampleLayerTests_smoke, Resample) {
 
     std::map<std::string, std::string> params;
     params["antialias"] = std::to_string((int)antialias);
+    params["coordTransMode"] = std::to_string((int)coordTransMode);
```

Comment:
What typical values can `coordTransMode` have? Are they numbers converted to strings? Or just names like "asymmetric"?